services/DataApiService.cfc

```diff
@@ -303,12 +303,18 @@ component {
 	}
 
 	private any function _renderField( required any value, required string renderer, struct fieldSettings={} ) {
+
+		var emptyNumericQuotes = false;
```

Comment:
I think we should be able to ditch these four lines (which could have been just one line anyway - `var emptyNumericQuotes = $isFeatureEnabled( "..." );`).

Just use the `$isFeatureEnabled()` call _only_ when dealing with the numeric type:

```cfc
case "numeric": 
    if ( IsNumeric( arguments.value ) ) {
        return arguments.value;
    }
    return $isFeatureEnabled( "..." ) ? NullValue() ?: "";
```

I also just spotted that this is now returning `true` or `false` instead of the number!